Problem with deploying website due to wasm-bindgen version

The following can be found in the CI log:

```console
 Compiling lsp-browser v0.0.0 (/home/runner/work/polarity/polarity/web/crates/browser)
    Finished `release` profile [optimized] target(s) in 1m 06s
error: 

it looks like the Rust project used to create this Wasm file was linked against
version of wasm-bindgen that uses a different bindgen format than this binary:

  rust Wasm file schema version: 0.2.93
     this binary schema version: 0.2.94

Currently the bindgen format is unstable enough that these two schema versions
must exactly match. You can accomplish this by either updating this binary or
the wasm-bindgen dependency in the Rust project.

You should be able to update the wasm-bindgen dependency with:

    cargo update -p wasm-bindgen --precise 0.2.94

don't forget to recompile your Wasm file! Alternatively, you can update the
binary with:

    cargo install -f wasm-bindgen-cli --version 0.2.93

if this warning fails to go away though and you're not sure what to do feel free
to open an issue at https://github.com/rustwasm/wasm-bindgen/issues!

make: *** [Makefile:13: build-server] Error 1
```

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -11,7 +11,7 @@ jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - run: |
           cd web/
           make deps

--- a/web/Makefile
+++ b/web/Makefile
@@ -1,6 +1,6 @@
 deps:
 	@rustup target add wasm32-unknown-unknown
-	@cargo install wasm-bindgen-cli
+	@cargo install -f wasm-bindgen-cli --version 0.2.93
 	@npm install
 
 build: build-server build-app

